src/SimpleWallet/SimpleWallet.cpp

```diff
@@ -1993,15 +2212,56 @@ bool simple_wallet::listTransfers(const std::vector<std::string>& args) {
   }
 
   if (!haveTransfers) {
-    success_msg_writer() << "No transfers";
+    successMsgWriter() << "No transfers";
   }
 
   return true;
 }
+//----------------------------------------------------------------------------------------------------
+bool simple_wallet::listMessages(const std::vector<std::string>& args) {
```

Comment:
Inside this function each tab character must be replaced with 4 spaces. Please, never use tabs for indentation!